velox/expression/CMakeLists.txt

```diff
@@ -21,8 +21,8 @@ add_library(
                              ReverseSignatureBinder.cpp)
 
 target_link_libraries(
-  velox_expression_functions velox_common_base velox_type_calculation
-  velox_type_signature velox_signature_parser)
+  velox_expression_functions velox_type_parser velox_common_base
```

Comment:
https://github.com/facebookincubator/velox/blob/main/velox/type/CMakeLists.txt#L23

Is adding it to `velox_type` a good idea? Add a new file named `NumberUtil.h`

Reply:
I feel it's ok as ParserUtil provides some helper functions on the handling of string.